client/login/magic-login/index.jsx

```diff
@@ -132,8 +133,20 @@ class MagicLogin extends Component {
 
 	renderLinks() {
 		const { isJetpackLogin, locale, showCheckYourEmail, translate } = this.props;
-
-		if ( showCheckYourEmail || this.props.query?.client_id ) {
+		const appPromo = (
+			<AppPromo
+				title={ translate( 'Stay logged in' ) }
+				campaign="emailed-login-link"
```

Comment:
Should be `source` right? 

```suggestion
				source="emailed-login-link"
```

Reply:
Sounds good!